README.rst

```diff
@@ -28,6 +28,8 @@ This repository includes shared utilities for:
 
 .. _Monitoring Utilities: edx_django_utils/monitoring/README.rst
 
+.. _Logging Utilities: edx_django_utils/logging/README.rst
```

Comment:
1. This adds a link, but you don't reference the link in the text above.
2. Should we use alphabetical order? We mostly do in the text, except the last section was probably just added to the end. Not sure why you decided to add this link in this order.

Reply:
Ah whoops, fixed! As for order, I just saw Monitoring and thought "hey, that's closest thematically, I'll put it after that". But now it's alphabetical, and right above Monitoring instead. :-)

Reply:
It's interesting that some doc linter didn't catch that. I bet it would have been caught in at least one of our projects.

Reply:
I would have moved the final bullet and link which are still out of order, but I won’t block on that.

Reply:
Ah, I can slip that in as well.